LoopKit/InsulinKit/InsulinMath.swift

```diff
@@ -301,7 +325,8 @@ extension DoseEntry {
         }
         
         let netTempBasalDoses = self.annotated(with: basalRateSchedule)
-        return netTempBasalDoses.glucoseEffects(insulinModel: insulinModel, insulinSensitivity: insulinSensitivity)
+        // Assume that temp basals will be delivered by the pump
+        return netTempBasalDoses.glucoseEffects(defaultModel: insulinModel, longestEffectDuration: insulinModel.effectDuration, insulinSensitivity: insulinSensitivity)
```

Comment:
Keeping track of "longestEffectDuration" ensures we have all the doses that could still be active